packages/app/components/profile/profile.tsx

```diff
@@ -232,7 +232,13 @@ const Profile = ({ username }: ProfileScreenProps) => {
                 <HeaderLeft canGoBack={true} withBackground />
               )
             }
-            headerRight={<HeaderDropdown type="settings" withBackground />}
+            headerRight={
+              <HeaderDropdown
+                type="settings"
+                withBackground
+                data={profileData?.data}
```

Comment:
nit: should we rename `data` to just `user`?

Reply:
Yes. Honestly thought that but being sick made me lazy lol.